build/images/scripts/restart_sandbox

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
according to what it actually does, it's better to name it something like migrate_cni